src/meta/mod.rs

```diff
@@ -248,3 +248,16 @@ impl Meta {
         })
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::Meta;
```

Comment:
We can probably move the `#[cfg(unix)]` thing above so that Github actions will not add this warning of unused import.